packages/widget_toolkit_pin/example/lib/main.dart

```diff
@@ -58,11 +58,6 @@ class MyHomePage extends StatelessWidget {
         ],
         child: Builder(
           builder: (context) => Scaffold(
-            appBar: AppBar(
-              title: Text(title),
```

Comment:
The `title` can be removed entirely from the `MyHomePage` since it's not used anymore

Reply:
Done